### PR TITLE
cIRI README file

### LINKED ISSUE #149: cIRI README

### PR DESCRIPTION
**Pull request by thibault-martinez**
This PR adds a README file for cIRI, fixes #149

### CODE REVIEW

File: ciri/README.md [hunk 1]
```diff
@@ -0,0 +1,61 @@
+cIRI
+====
+
+A low level implementation of a IOTA node in C.
```
Comment (th0br0) [inline ciri/README.md]: `an IOTA node`

File: ciri/README.md [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+
+This is a full-featured [IOTA](https://iota.org/) node with a convenient JSON-REST HTTP interface. It allows users to become part of the [IOTA](https://iota.org/) network as both a transaction relay and network information provider through the easy-to-use [API](https://iota.readme.io/reference).
```
Comment (th0br0) [inline ciri/README.md]: That's not really the case today? Let's call it WIP node?

File: ciri/README.md [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+
+It is specially designed for users seeking a fast, efficient and fully-compatible network setup.
```
Comment (th0br0) [inline ciri/README.md]: compatible with what?

File: ciri/README.md [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+
+Running an IRI node also allows light wallet users a node to directly connect to for their own wallet transactions.
```
Comment (th0br0) [inline ciri/README.md]: let's remove this for now

File: ciri/README.md [hunk 5]
```diff
@@ -0,0 +1,61 @@
...
+- As a 'friend-to-friend' network, you have the privilege of joining new users into the network through your node by adding them to your approved neighbors list — ensuring that you both broadcast to them and also receive their broadcasts.
+
+You can find neighbors quickly at both our [Discord Community](https://discord.gg/7Gu2mG5) and [Forum](https://forum.iota.org/).
+
+Everyone will be welcoming and very happy to help you get connected. If you want to get tokens for your testcase, please just ask in one of the communication channels.
+
+Build and run cIRI
+------------------
+
+cIRI is built and ran through [bazel](https://www.bazel.build/).
```
Comment (th0br0) [inline ciri/README.md]: `and run`

File: ciri/README.md [hunk 6]
```diff
@@ -0,0 +1,61 @@
...
+
+```
+$ git clone https://github.com/iotaledger/entangled.git
+$ cd entangled
+$ bazel run -- ciri
```
Comment (th0br0) [inline ciri/README.md]: `bazel run -- ciri <optional flags>`

File: ciri/README.md [hunk 7]
```diff
@@ -0,0 +1,61 @@
...
+Long option | Short option | Description | Example input
+--- | --- | --- | ---
+`--help` | `-h` | Displays the usage |
+`--log-level` | `-l` | Log level amongst: "debug", "info", "notice", "warning", "error", "critical", "alert" and "emergency" | `-l debug`
+`--neighbors` | `-n` | Neighbors that you are connected with will be added via this option | `-n "udp://148.148.148.148:14265 udp://[2001:db8:a0b:12f0::1]:14265"`
+`--udp-receiver-port` | `-u` | UDP receiver port | `-u 14600`
+`--tcp-receiver-port` | `-t` | TCP receiver port | `-t 15600`
+`--testnet` | | Makes it possible to run IRI with the IOTA testnet | `--testnet`
+
+TODO
```
Comment (th0br0) [inline ciri/README.md]: let's rather link to some parent tracking issue at the top of this readme instead of having to update this document regularly?

**Review by th0br0 — CHANGES_REQUESTED** [review]

File: ciri/README.md (hunk 4 above)
Comment (tsvisabo) [inline ciri/README.md]: `allows light wallet users a node to directly connect to for` ->
`allows light wallet users to have a node with whom they can directly connect to for`
both works, whichever..

**Review by tsvisabo — CHANGES_REQUESTED** [review]

File: ciri/README.md [hunk 8]
```diff
@@ -25,12 +28,12 @@ Everyone will be welcoming and very happy to help you get connected. If you want
...
 ------------------
 
-cIRI is built and ran through [bazel](https://www.bazel.build/).
+cIRI is built and run through [bazel](https://www.bazel.build/).
 
 ```
 $ git clone https://github.com/iotaledger/entangled.git
 $ cd entangled
-$ bazel run -- ciri
+$ bazel run -- ciri <optional flags>
```
Comment (tsvisabo) [inline ciri/README.md:36]: This should also contain a table for more detailed configuration usage

**Review by tsvisabo — CHANGES_REQUESTED** [review]

File: ciri/README.md (hunk 8 above)
Comment (th0br0) [reply ciri/README.md:36]: It does, see below

**Review by th0br0 — APPROVED** [review]

**Review by tsvisabo — APPROVED** [review]

**Pull request merged**